app/components/errors/special-error-handler.tsx

```diff
@@ -0,0 +1,57 @@
+import { tw } from "~/utils/tw";
+import { SpecialErrorAdditionalData } from "./utils";
+import { useMemo } from "react";
+import { ErrorIcon } from ".";
+import { Button } from "../shared/button";
+
+export type SpecialErrorHandlerProps = {
+  className?: string;
+  style?: React.CSSProperties;
+  additionalData: SpecialErrorAdditionalData;
+};
+
+export default function SpecialErrorHandler({
+  className,
+  style,
+  additionalData,
+}: SpecialErrorHandlerProps) {
+  const content = useMemo(() => {
+    switch (additionalData.type) {
+      case "asset-from-other-org": {
```

Comment:
I think we need to make this more dynamic. `Entity from other mode` and then we can pass the model so `asset`, `kit`, `category...etc.